static/app/views/starfish/views/spans/spanTimeCharts.tsx

```diff
@@ -108,10 +124,17 @@ function ThroughputChart({moduleName, filters}: ChartProps): JSX.Element {
   const throughputTimeSeries = Object.keys(dataByGroup).map(groupName => {
     const groupData = dataByGroup[groupName];
 
+    let throughputMultiplier = 1; // We're fetching per minute, so default is 1
+    if (throughtputUnit === RateUnits.PER_SECOND) {
+      throughputMultiplier = 60;
```

Comment:
I got this mixed up, should be divde by 60 to convert per minute to per second.

Reply:
I'm going to put this in draft for now, and fix the units everywhere, I think this might be a bigger problem

Reply:
All Discover responses come back with a `meta` field that describes the units, is it possible to rely on that, instead? That's what all the tables do, and the charts should, too, if possible

Reply:
Yeah I really like that idea, but I would want to keep a change like that outside the scope of this PR if that works.

If I understand correctly, for the resource module we can declare one variable for the throughput unit, that would dictate what we request for (sps(), spm(), sph(), etc), and then the charts just read off of meta to display the right stuff. We would probs wanna do this generically (i.e pass in meta to the chart).

The only gotcha here is that we have to add backend support for any unit of measurement we need, rather then the client converting it. Also Imo, the unit we use to display is more of a client concern, so I think some might argue that we should keep the transformation in the client (I'm neutral against that argument)